src/test/java/io/github/perrymant/moneymaker/ApplicationTest.java

```diff
@@ -44,21 +63,63 @@
             "        ╚════════════╧══════════════════╧════════╧═════════╧═════════════╝\n";
 
     @Test
-    public void printsHelpMessageIfHelpIsGiven() {
+    public void noArgumentsIsGiven_printsError() {
+        target.start(new String[]{});
```

Comment:
we've got this line in several places. Let's do all the assertions for this case in one test. Then it might be easier to spot if all the assertions should even be there. I think there might be some redundant ones around right now.

Reply:
- I think I cleaned up some tests so I think all tests are valid now.
- I'm not sure how to put all assertions in one test. would it look like many of these in the same test?:
```
Assert.assertEquals(<expected>, <actual>);
Assert.assertEquals(<expected>, <actual>);
Assert.assertEquals(<expected>, <actual>);
Assert.assertEquals(<expected>, <actual>);
```

Reply:
Yes. Multiple assertions like this is very common. As long as they are validating different things. Tests are looking good as they are right now.

Reply:
Hmmm. I've definitely seen this "all assertions in one test" in other places, but I can't see how it will be done here... - I tried to make a loop that goes through an array of all the test input (at the moment it's only got the first three). Is this perhaps a way to go forward?